test/thermodynamics/inner_energy_from_sum_heat_and_work_test.py

```diff
@@ -0,0 +1,41 @@
+from collections import namedtuple
+from pytest import approx, fixture, raises
+from symplyphysics import (
+    errors,
+    units,
+    Quantity,
+    SI,
+    convert_to,
+)
+from symplyphysics.laws.thermodynamics import inner_energy_from_sum_heat_and_work as first_law_of_term
+
+
+@fixture(name="test_args")
+def fixture_test_args():
+    W = Quantity(50 * units.joule)
+    Q = Quantity(100 * units.joule)
+    Args = namedtuple("Args", ["Q", "W"])
+    return Args(Q=Q, W=W)
+
+
+def test_basic_delta_inner_energy(test_args):
+    result = first_law_of_term.calculate_inner_energy(test_args.Q, test_args.W)
+    assert SI.get_dimension_system().equivalent_dims(result.dimension, units.energy)
+    result_delta_inner_energy = convert_to(result, units.joule).evalf(2)
+    assert result_delta_inner_energy == approx(150, 0.01)
```

Comment:
So, according to the description, system has got 100 joules, done work of 50 joules and now it's inner energy is 150 joules? Use some external example to verify your law.